data/ledgerentry.go

```diff
@@ -133,12 +133,16 @@ type Escrow struct {
 	DestinationNode *NodeIndex       `json:",omitempty"`
 }
 
-type SignerEntry struct {
+type SignerEntryItem struct {
```

Comment:
Have you got an example tx or ledge entry from mainnet which demonstrates this structure?

Reply:
* Doc: https://xrpl.org/signerlistset.html#example-signerlistset-json
* Here is the tesnet example (unfortunately I don't have the same for the mainnet): https://testnet.xrpl.org/transactions/01A6D400C0F114543A3BE26708F9E3CBBFDDDF40000E1D5C85E3F0DDFE5EC9C1/raw

Reply:
My bad, that was the proof for the  `Signer` and `SignerItem`.
And here is the proof for the `SignerListSet`: https://testnet.xrpl.org/transactions/05AADAF31482E06241870D93D32FDC5E59414D7A9D60754D1C1DEC5ACFDF3860/raw